### PR TITLE
add new field alternatives, add new methods addAlternative, removeAlt…

### PR DESCRIPTION
**Pull request by aloupfor**
…ernative, getAlternatives

https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/112

### CODE REVIEW

File: MediaModelBundle/Document/Media.php [hunk 1]
```diff
@@ -314,6 +319,35 @@ public function getUsageReference()
...
+    {
+        $this->alternatives[$formatName] = $alternativeName;
+    }
+
+    /**
+     * @param string $formatName
+     */
+    public function removeAlternative($formatName)
+    {
+        foreach ($this->alternatives as $key) {
```
Comment (itkg-alavieille) [inline MediaModelBundle/Document/Media.php]: i don't understand why you use a foreach. 
Why not just : 
```
if (isset($this->alternatives[$formatName])) {
   unset($this->alternatives[$formatName]);
}
```

File: Media/Model/MediaInterface.php [hunk 2]
```diff
@@ -143,6 +143,22 @@ public function removeUsageReference($reference);
...
+
+    /**
+     * @param string $formatName
+     */
+    public function removeAlternative($formatName);
+
+    /**
+     * @return array
+     */
+    public function getAlternatives();
```
Comment (itkg-ngilain) [inline Media/Model/MediaInterface.php]: Maybe you can add a getAlternative($formatName), it seems usefull for an integrator, but I don't know if we will trully need it

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**